backend/src/models/pathway.ts

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+
+const pathwaySchema = new mongoose.Schema({
```

Comment:
Have you looked into using the typescript interfaces with the mongoose schema. I think its more effort than its worth (especially for right now), but wanted to ask anyway.
https://medium.com/@tomanagle/strongly-typed-models-with-mongoose-and-typescript-7bc2f7197722

Reply:
I stumbled upon this while investigating the mongoose types issues, but I didn't take a serious look at it.  If we start doing some more serious business logic with the pathways or criteria in the backend then I think it would make sense.